adaptive-backend/internal/models/llms.go

```
@@ -24,6 +28,232 @@
+	// [model](https://platform.openai.com/docs/models) you use, different message
+	// types (modalities) are supported, like
+	// [text](https://platform.openai.com/docs/guides/text-generation),
+	// [images](https://platform.openai.com/docs/guides/vision), and
+	// [audio](https://platform.openai.com/docs/guides/audio).
+	Messages []openai.ChatCompletionMessageParamUnion `json:"messages,omitzero"`
+	// Model ID used to generate the response, like `gpt-4o` or `o3`. OpenAI offers a
+	// wide range of models with different capabilities, performance characteristics,
+	// and price points. Refer to the
+	// [model guide](https://platform.openai.com/docs/models) to browse and compare
+	// available models.
+	Model shared.ChatModel `json:"model,omitzere"`
```

golangci-lint (1.64.8)

SA5008: unknown JSON option "omitzere"

(staticcheck)